HOW_WE_WORK.md

```diff
@@ -0,0 +1,13 @@
+# How We Work
```

Comment:
This is an outdated duplicate of [this doc](https://github.com/csg-org/CompactConnect/blob/development/docs/HOW_WE_WORK.md). We should probably remove it.

Reply:
I don't think that's quite right, but I may be wrong. #115 was merged into main, rather than development. This document includes the per-sprint language we discussed in that PR, while the one you linked does not.